tests/functional/aws-node-sdk/test/multipleBackend/put.js

```diff
@@ -51,7 +51,7 @@ function awsGetCheck(objectKey, s3MD5, awsMD5, cb) {
             if (process.env.ENABLE_KMS_ENCRYPTION === 'true') {
                 // doesn't check ETag because it's different
```

Comment:
Just curious, you mean the returned ETag doesn't match the MD5 sum of the actual object?